### ISSUE TITLE
Make map marker update in own callback

### ISSUE THREAD

**Issue by drewmresnick** [iridl/python-maprooms#168]
Unrelated to Drew's changes, I notice that when you click on the map, the location of the marker doesn't get updated until the graphs finish rendering. Could we update the marker in a separate callback instead of piggybacking it on `local_plots`?

_Originally posted by @aaron-kaplan in https://github.com/iridl/python-maprooms/issues/156#issuecomment-1241179126_

**Comment by remicousin** [comment]
For Onset Maproom, do the snapping of the selection first:
https://github.com/iridl/python-maprooms/issues/144

**Comment by remicousin** [comment]
I am going to look into that for forecast now.

**Comment by remicousin** [comment]
I don't know if this is actually possible because it's a 3-band game. User can update graphs by either clicking map or submitting lat/lon. But also clicking map must update lat/lon inputs and submitting lat/lon must update Marker and themselves as the selection goes to the center of the grid.

and Dash is telling me: "Any given output can only have one callback that sets it." So right now I don't see how this all can be done with more than 1 unique callback...

**Comment by aaron-kaplan** [comment]
@remicousin I don't see a loop in the data flow you've described, so it should be possible to break it up.

- One callback takes map click and lat/lon text inputs (the latter as State rather than Input, which is already the case) and outputs marker location and lat/lon inputs.
- Another callback takes marker location and outputs the graph.

BTW the current callback has
```
    Output("map", "click_lat_lng"),
```
What was that intended to do? I don't think setting this property has any effect. It's only meant to be used as an Input, not an Output.

**Comment by remicousin** [comment]
So if a user uses the text inputs to submit a location, that will update the Marker location and that would cascade to trigger the callback that outputs the graphs?

with regards to the BTW: I just notices that this guy was both an Input and an Output. I will make sure to clean that up when addressing the issue here.

**Comment by aaron-kaplan** [comment]
> So if a user uses the text inputs to submit a location, that will update the Marker location and that would cascade to trigger the callback that outputs the graphs?

Yes. Probably a bit slower because there are two round-trips to the server instead of one, but I feel like it's worth it to have the quick feedback indicating that the click has taken effect.

**Comment by remicousin** [comment]
Good idea to assign rather than send email to notify when someone takes over something

**Comment by remicousin** [comment]
PR #185 does it for forecast. Now can port that plus snapping to onset.
